The-SMIRNOFF-force-field-format.md

```diff
@@ -107,21 +292,30 @@ In the future, we may switch to a model where torsional barriers are [automatica
 This would result in better handling of some cases where a small change in a molecule (such as a change in tautomer) could currently (as in AMBER) result in a dramatically different applied barrier height because of a change in the number of torsions passing through that bond.
 The averaging approach would make it easier to avoid this problem without requiring as many different torsional terms.
 
-### IMPROPER TORSIONS
+### `<ImproperTorsionForce>`
 
-Impropers are applied in the same manner as proper torsions, via `PeriodicTorsionForce`, but with the `Improper` tag, as in:
+Improper torsions are specified via an `<ImproperTorsionForce>...</ImproperTorsionForce>` block, with individual `<Improper/>` tags containing attributes that specify the same properties as `<ProperTorsionForce>`:
 ```XML
-<Improper smirks="[*:1]~[#6X3:2](=[#7X2,#7X3+1:3])~[#7:4]" k1="10.5" periodicity1="2" phase1="180."/>
-...
+<ImproperTorsionForce potential="charmm" phase_unit="degrees" k_unit="kilocalories_per_mole">
+    <Improper smirks="[*:1]~[#6X3:2](=[#7X2,#7X3+1:3])~[#7:4]" k1="10.5" periodicity1="2" phase1="180."/>
+    ...
+</ImproperTorsionForce>
+```
+Currently, only `potential="charmm"` is supported, where we utilize the functional form:
 ```
+U = \sum_{i=1}^N k_i * (1 + cos(periodicity_i * phi - phase_i))
+```
+**Note that AMBER defines a modified functional form**, such that `U = \sum_{i=1}^N (k_i/2) * (1 + cos(periodicity_i * phi - phase_i))`, so that barrier heights would need to be divided by two in order to be used in the SMIRNOFF format.
+If the `potential` attribute is omitted, it defaults to `charmm`.
 
-**Improper torsions deviate profoundly from AMBER handling of impropers** in two ways.
+**Improper torsion definitions deviate profoundly from AMBER handling of impropers** in two ways.
 First, to eliminate ambiguity, we treat impropers as a [trefoil](https://upload.wikimedia.org/wikipedia/commons/thumb/5/5c/Trefoil_knot_left.svg/2000px-Trefoil_knot_left.svg.png) and apply the same set of parameters to all six paths around the trefoil.
-*Because of this, all barrier heights are divided by six before we apply them*, for consistency with AMBER force fields. Second, the *second* atom in an improper (in the example above, the trivalent carbon) is the central atom in the trefoil.
+*Because of this, all barrier heights are divided by six before we apply them*, for consistency with AMBER force fields.
+Second, the *second* atom in an improper (in the example above, the trivalent carbon) is the central atom in the trefoil.
```

Comment:
These two lines together are confusing to me. I think they are making separate points, but they are close together, so I'm trying to interpret them together.

First you say the the "improper definitions deviate profoundly" (which I understand), and I understand the point about a trefoil, I think, but then you say you do something "for consistency with AMBER". So what part is still profoundly different? I think this deserves some more clarity.

Reply:
Maybe we need to explicitly say that AMBER only applies an improper to one particular path through the atoms, creating impropers that depend on the order of atoms in one's input.

Reply:
That would be good. Of course it's not really the job *this* specification to go into detail on all the other specifications, but anything that will help an AMBER (or other) user quickly be able to think about their system in SMIRNOFF format is useful and helpful. In this case, is their guidance for how to translate an AMBER improper line to a SMIRNOFF one?

Reply:
> is their guidance for how to translate an AMBER improper line to a SMIRNOFF one?

Heh, @slochower -- "no". You could also ask whether there is any guidance as to how to translate an AMBER improper line into an AMBER improper line which is equivalent on reshuffling the order of the input atoms, and the short answer is "no". The long answer is "nooo". 

Our current approach results in relatively similar energies to AMBER for many geometries but removes the atom order dependence that is present in AMBER, so that reshuffling atoms doesn't change the details of the improper which is applied. But the AMBER behavior is sort of a type of "bug" or "feature with unpleasant behavior" and there is no way we are going to  exactly reproduce it.

So, guidance is that to get nearly the same behavior (but not identical) as AMBER, you just port in the AMBER parameters directly.